gaei_cpp/wall.hpp

```diff
@@ -0,0 +1,47 @@
+﻿#pragma once
+#include<iostream>
+#include<vector>
+#include"vertex.hpp"
+
+namespace gaei {
+inline void bounding_box(std::vector<gaei::vertex<gaei::vec3f, gaei::color>>& vertexes) {
+    typename vec3f::value_type max_x, max_y, min_x, min_y, min_z;
+    max_x = vertexes[0].position.x();
+    min_x = vertexes[0].position.x();
+    max_y = vertexes[0].position.y();
+    min_y = vertexes[0].position.y();
+    min_z = vertexes[0].position.z();
+    for (int i = 0; i < vertexes.size(); i++) {
```

Comment:
無駄な比較と`signed`と`unsigned`の比較がある。
```suggestion
    for (unsigned i = 1; i < vertexes.size(); i++) {
```